From adf3fefc4afa6225afedaf5f92b0501dafa99c65 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Diego=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Sat, 30 Aug 2008 14:51:46 +0200
Subject: [PATCH] Fix undefined references for XStringToKeysym.

The XStringToKeysym function comes from libX11, link to it to not have
undefined references in the shared library.
---
 configure.ac    |    2 ++
 src/Makefile.am |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1668771..b8bc4cf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -91,6 +91,8 @@ AC_SUBST(plugindir)
 PKG_CHECK_MODULES(LIBXML2, "libxml-2.0")
 AC_SUBST(LIBXML2_REQUIRES)
 
+PKG_CHECK_MODULES(X11, x11)
+
 AC_CHECK_HEADERS([sys/inotify.h], [have_inotify=yes], [have_inotify=no])
 
 globalmetadata=$datadir/compizconfig/global.xml
diff --git a/src/Makefile.am b/src/Makefile.am
index ac7424f..9bd9722 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -22,7 +22,7 @@ libcompizconfig_la_SOURCES = \
 	ccs-private.h	\
 	iniparser.h
 
-libcompizconfig_la_LIBADD = @LIBXML2_LIBS@
+libcompizconfig_la_LIBADD = @LIBXML2_LIBS@ $(X11_LIBS)
 
 lib_LTLIBRARIES=libcompizconfig.la
 
-- 
1.6.0.1

